src/app/app.component.ts

```diff
@@ -1,10 +1,77 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { discardPeriodicTasks } from '@angular/core/testing';
+import { Injectable, Pipe, PipeTransform } from '@angular/core';
+import { SearchFilterPipe } from './filter.pipe';
+import { Task } from './task/task.component';
+
 
 @Component({
   selector: 'app-root',
+  styleUrls: ['./app.component.css'],
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.css']
 })
-export class AppComponent {
-  title = 'app';
-}
+
+export class AppComponent implements OnInit {
+  tasks: Task[] = [];
+  newTask: string = '';
+  id = 1;
+  itemFilter: any = { text: '' };
+  tasksDone: any = [];
+  title: string;
+
+  ngOnInit() {
+    this.title = '';
+    this.tasks = [];
```

Comment:
Na to już zwracałem uwagę...

Reply:
https://github.com/jedi800/Checklistaa/pull/1/commits/b2b1d80ca0a1b7bcf43949af1d402dd11741f093